json-endpoints/class.wpcom-json-api-list-posts-endpoint.php

```diff
@@ -153,7 +153,7 @@ function callback( $path = '', $blog_id = 0 ) {
 			'post_status'    => $status,
 			'post_parent'    => isset( $args['parent_id'] ) ? $args['parent_id'] : null,
 			'author'         => isset( $args['author'] ) && 0 < $args['author'] ? $args['author'] : null,
-			's'              => isset( $args['search'] ) ? $args['search'] : null,
+			's'              => isset( $args['search'] ) && '' !== $args['search'] ? $args['search'] : null,
```

Comment:
Not a blocker: could we just use `! empty()` rather than adding an operator?

Reply:
`'0'` is empty, so I decided to go with this. Searching for `'0'` is probably broken in other ways, though :)